makefiles/shared.mk

```diff
@@ -77,9 +77,14 @@ next-gen-stage: ## Host online for review
 	fi
 endif
 
+ifeq ($(MUT_PREFIX),)
```

Comment:
Could we have a comment explaining the purpose of this condition?

Reply:
Absolutely! Thanks